src/runtime/pkg/device/drivers/vfio.go

```diff
@@ -29,10 +29,10 @@ const (
 	vfioRemoveIDPath    = "/sys/bus/pci/drivers/vfio-pci/remove_id"
 	iommuGroupPath      = "/sys/bus/pci/devices/%s/iommu_group"
 	vfioDevPath         = "/dev/vfio/%s"
-	pcieRootPortPrefix  = "rp"
 )
 
 var (
+	// AllPCIeDevs keeps track of all PCIe devices
```

Comment:
The comment does not add much compared to the name.

That made me wonder what this was used for, and the only reads I can find of the structure are in the form `len(AllPCIeDevs)`. So this means we are just counting the PCIe devices. Unless we end up hot-plugging the same thing multiple times, I do not really understand what a `map` buys us here. Any idea?

Reply:
Indeed the comment is not helpful. It is used to deduce the correct bus number for the root or switch port. The BDF as a key will make sure that we do no increase the bus number if we hit the very same device. If e.g. someone pass throughs the very same device twice.